tests/e2e/api/controllers/_changes.spec.js

```diff
@@ -389,33 +386,10 @@ describe('changes handler', () => {
   });
 
   describe('Filtered replication', () => {
-    const couchVersionForBatching = '2.3.0';
 
-    let shouldBatchChangesRequests;
     let bobsIds;
     let stevesIds;
 
-    before(() => {
-      const options = {
-        hostname: constants.COUCH_HOST,
-        port: constants.COUCH_PORT,
-        auth: auth.username+ ':' + auth.password,
-        path: '/'
-      };
-
-      const req = http.request(options, res => {
-        let body = '';
-        res.on('data', data => body += data);
-        res.on('end', () => {
-          body = JSON.parse(body);
-          shouldBatchChangesRequests = semver.lte(couchVersionForBatching, body.version);
-        });
-      });
-
-      req.on('error', e => e);
-      req.end();
-    });
-
```

Comment:
Dropped due to direct Couch access being blocked. This could be refactored to use another method but I don't think we need to support couch 2.3.0 any more do we?

Reply:
Ok. I've removed the associated production code too!